src/meta/Matches.ts

```diff
@@ -0,0 +1,14 @@
+import { Base } from './Base';
+
+export default class Matches extends Base {
+	/**
+	 * Determine if the target of a particular `Event` matches the virtual DOM key
+	 * @param key The virtual DOM key
+	 * @param event The event object
+	 */
+	public get(key: string, event: Event): boolean {
+		this.requireNode(key);
```

Comment:
Adding the key as a required node is a bit confusing to me. Do you think that when used in an event handler, if the `node1` has not been rendered that it should 1) cause an immediate re-render or 2) throw an error if the node doesn't still doesn't exist in the next render? of the back of a user action?

```ts
myEventHandler(event: Event) {
    if (this.meta(Matches).get('node1', event)) {
        // do something 
    }
    else if (this.meta(Matches).get('node2', event)) {
        // do something         
    }
    else if (this.meta(Matches).get('node3', event)) {
        // do something 
    }
    else if (this.meta(Matches).get('node4', event)) {
        // do something         
    }
}
```